### DISCUSSION TITLE
./run.sh code error

### DISCUSSION THREAD

**Discussion by locci** [pmd/pmd#2991] [Q&A] [answered]
Hi. 

I am new in the neighborhood, so sorry if my question is silly.
Well, the problem is I build the pmd as explained in this site "https://pmd.github.io/latest/pmd_devdocs_building.html" and all goes fine until I try to run the file  ruin.sh. The error message is  

alexandre@alexandre-VirtualBox:~/pmd-bin-6.30.0/bin$ sh run.sh 
run.sh: 147: Syntax error: "(" unexpected

but the code looks like right to me.

**Comment by oowekyala** [comment, marked as answer]
The script needs bash, it should work if you just use `bash run.sh`, or `./run.sh`

**Reply by adangel** [reply]
yes, either use bash or better just run ./run.sh - the script is executable.
